### ISSUE TITLE
allow inline type declaration in application/json's type

### ISSUE THREAD

**Issue by iwanbk** [Jumpscale/go-raml#231]
example:
```
 post:
        displayName: Create a new CoreX
        body:
          application/json:
            type: [ Command, CoreX ]

```

**Comment by iwanbk** [comment]
we currently need to define new type and use it in application/json type

**Comment by iwanbk** [comment]
Go support has been implemented in #246 

Nim & Python support will be part of #245

**Comment by iwanbk** [comment]
Nim has been implemented in #245 while python is not.

Closing this issue because python issue is not about detecting the inline type declarations but about the python's support for advanced type (#256)
